### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by weiweihuanghuang** [Mark2Mark/Show-Smart-Plumblines#2]
**Draw line between two points**
This would be great to see how diagonal lines extend

**Toggle**
I noticed you had code in there for bounding box lines, they are great too, would be nice to have a toggle for that too?

**Comment by Mark2Mark** [comment]
**Re: Draw line between two points**
you mean just between them, or extending both points a little bit?

**Re: Toggle**
Indeed, also toggling is relatively easy now. I just struggled to render a bounding box according to an italic angle. Somehow this is more complex than I thought. The BB itself can be usefull both: angled and straight. What do you think?

**Comment by weiweihuanghuang** [comment]
> you mean just between them, or extending both points a little bit?

Extending out to the screen edge actually — to be honest maybe not that useful, I was just thinking of situations where I want to see if a diagonal line actually goes in a certain direction to meet a line, but actually it's better to judge it optically.

> Indeed, also toggling is relatively easy now. I just struggled to render a bounding box according to an italic angle. Somehow this is more complex than I thought. The BB itself can be usefull both: angled and straight. What do you think?

Angled is an italic box right?

**Comment by Mark2Mark** [comment]
Right. BB by default is always a rectangle. In italics, I sometimes find this an unsufficient is an information. Will see what I can do.
